lerobot/common/robot_devices/motors/dynamixel.py

```diff
@@ -127,18 +140,19 @@
     "xl430-w250": X_SERIES_BAUDRATE_TABLE,
     "xm430-w350": X_SERIES_BAUDRATE_TABLE,
     "xm540-w270": X_SERIES_BAUDRATE_TABLE,
+    "xc430-w150": X_SERIES_BAUDRATE_TABLE,
 }
 
 NUM_READ_RETRY = 10
 NUM_WRITE_RETRY = 10
 
 
-def convert_degrees_to_steps(degrees: float | np.ndarray, models: str | list[str]):
+def convert_degrees_to_steps(degrees: int | float | np.ndarray, models: str | list[str]):
     """This function convert the degree range to the step range for indicating motors rotation.
     It assums a motor achieves a full rotation by going from -180 degree position to +180.
     The motor resolution (e.g. 4096) corresponds to the number of steps needed to achieve a full rotation.
     """
-    if isinstance(degrees, float):
+    if isinstance(degrees, (float, int)):
```

Comment:
nit: why is this needed? I think you can just leave the type hint as a float and not convert to a numpy object here.

Reply:
I'm happy to resolve but just FYI I think `degrees: float` serves the purpose without the other two. I normally opt for a minimal type hinting strategy.